tests/testthat/test-install.R

```diff
@@ -2,6 +2,7 @@ context("install")
 
 test_that("install_cmdstan() successfully installs cmdstan", {
   skip_if_offline()
+  if (os_is_windows()) skip_on_covr()
```

Comment:
In codecov Windows tests the install tests fail as if g++ can not be found (even though cmdstan is successfully built beforehand). This seems like a weird codecov thing that is not suitable for our install use case, hence I added skips here.